extend_gripper_control_xarm/scripts/gripper_control_inspire_v2.py

```diff
@@ -115,7 +115,8 @@ def SplitDecimal(decimal):
     #Configure the Baudrate for the tool modbus
     gripper_baudrate_service = rospy.ServiceProxy(configToolModbusServiceName, xarm_msgs.srv.ConfigToolModbus)
     gripper_baudrate_config = ConfigToolModbusRequest()
-    gripper_baudrate_config.baud_rate = 115200  #Baudrate for the Robotiq grippers
+    #gripper_baudrate_config.baud_rate = 115200  #Baudrate for the gripper
+    gripper_baudrate_config.baud_rate = int(os.getenv('gripperBaudrate', '115200'))
```

Comment:
Would it be useful to let the user (or at least the logs) know if this value hasn't been set?
How about something like

```suggestion
default_gripper_baud = 115200
try:
    gripper_baud = int(os.getenv('gripperBaudrate'))
except TypeError:
    rospy.logwarn(f"No 'gripperBaudrate' env var found, setting to default: {default_gripper_baud}")
    gripper_baud = default_gripper_baud
else:
    rospy.loginfo(f"'gripperBaudrate' env var found, setting baud to: {gripper_baud}")
    
gripper_baudrate_config.baud_rate = gripper_baud
```